packages/vue/src/examples/pages/error/Error.vue

```diff
@@ -0,0 +1,67 @@
+<template>
+  <div id="error">
+    <SfImage
+      class="image"
+      :src="{
+        mobile: { url: '/assets/storybook/Error/error_mobile.svg' },
+        desktop: { url: '/assets/storybook/Error/error.svg' }
+      }"
+    />
+    <SfHeading
+      title="Page not found"
+      subtitle="We are sorry that we can’t find the page, please go back or try again"
+      class="heading sf-heading--no-underline"
+    />
+    <div class="actions">
+      <SfButton class="sf-button--full-width actions__button">
+        Return to home
+      </SfButton>
+      <SfButton
+        class="sf-button--full-width sf-button--text actions__button"
+        @click="$emit('click:back')"
+      >
+        Back
+      </SfButton>
+    </div>
+  </div>
+</template>
+
+<script>
+import { SfButton, SfImage, SfHeading } from "@storefront-ui/vue";
+
+export default {
+  name: "Error",
+  components: { SfButton, SfImage, SfHeading }
+};
+</script>
+
+<style lang="scss" scoped>
+@import "~@storefront-ui/vue/styles";
+
+#error {
+  box-sizing: border-box;
+  padding: 0 1.25rem;
```

Comment:
```suggestion
  padding: 0 var(--spacer-big);
```

Reply:
can we use height:100% and center content on desktop and space-between on mobile?
![image](https://user-images.githubusercontent.com/12138170/75244950-0ad70f80-57cd-11ea-8595-51ce4b2dbe56.png)